src/module_wrap.h

```diff
@@ -29,11 +24,6 @@ enum HostDefinedOptions : int {
   kLength = 10,
 };
 
-v8::Maybe<url::URL> Resolve(Environment* env,
```

Comment:
was `PackageMainCheck` and `Resolve` just vestigial?

Reply:
/cc @guybedford who may know more

Reply:
Resolve is an internal function, so doesn't need to be in the header file, although we could certainly add it back if other C++ files need to use it.

PackageMainCheck was part of the previous Resolve API which is no longer used.

Reply:
I'd say remove it until we use it elsewhere in the codebase?

Reply:
Can this conversation be resolved or is there something actionable?